tools/c7n_azure/c7n_azure/function_binding_resources/readme.md

```diff
@@ -0,0 +1,8 @@
+## Storage Queue Binding Support for Azure Functions
+
+Azure functions storage queue binding support requires installing the dotnet storage queue extension. In order to publish this with the function the dlls need to be added locally. This directory contains the necessary files for the binding support to be published with the Azure Function.
+
+To update the files ensure [dotnet is installed](https://www.microsoft.com/net/download) on your local machine. Then run the following command:
+
+`dotnet build tools/c7n_azure/c7n_azure/function_binding_resources/extensions.csproj --configuration Release --output bin`
```

Comment:
I think its better to split it to:
cd tools/c7n_azure/c7n_azure_function_binding_resources
dotnet build extensions.csproj --configuration Release --output bin

With current command, you assume user in the root folder & as a result it creates bin folder in the root.

Reply:
I'll add a comment to run this command from root.